cmd/ingress/main.go

```diff
@@ -182,5 +201,21 @@ func (env *envConfig) send(event *cloudevents.Event, span *trace.Span) (int, err
 	if !ack {
 		return http.StatusInternalServerError, errors.New("failed to publish message: nacked")
 	}
+
 	return http.StatusAccepted, nil
 }
+
+func (env *envConfig) CreateRabbitMQConnections(
+	rmqHelper *rabbit.RabbitMQHelper,
+	retryChannel chan<- bool,
+	logger *zap.SugaredLogger) (wabbit.Conn, wabbit.Channel, error) {
+	conn, channel, err := rmqHelper.SetupRabbitMQ(env.BrokerURL, retryChannel, logger)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if err := env.channel.Confirm(false); err != nil {
+		logger.Errorw("Failed to switch connection channel to confirm mode: %s", zap.Error(err))
```

Comment:
It seems we used to do a `Fatalf` and fail the pod in this error case but now it's getting silently ignored.